.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:

```

Comment:
thanks its very good job, but you have to add this code `
int a = 5;
int y = a*12; 
if(y>50) console.log("good work")
`